src/debug.c

```diff
@@ -112,11 +112,10 @@ void xorStringObjectDigest(unsigned char *digest, robj *o) {
  * Also note that mixdigest("foo") followed by mixdigest("bar")
  * will lead to a different digest compared to "fo", "obar".
  */
-void mixDigest(unsigned char *digest, void *ptr, size_t len) {
+void mixDigest(unsigned char *digest, const void *ptr, size_t len) {
     SHA1_CTX ctx;
-    char *s = ptr;
 
-    xorDigest(digest,s,len);
+    xorDigest(digest,ptr,len);
```

Comment:
function xorDigest need changed too
```c
void xorDigest(unsigned char *digest, void *ptr, size_t len)
```

Reply:
@zeekling i don't understand what you mean. please be a bit more verbose.

Reply:
the args `ptr` in xorDigest is not const,Is there a problem？

Reply:
I am sure i understand your point, but `ptr` in xorDigest already is changed into `const` in my first commit.

Reply:
ok